packages/alfa-css/src/value/shape/rectangle.ts

```diff
@@ -0,0 +1,150 @@
+import { Value } from "../../value";
+import { Length } from "../length";
+import { Keyword } from "../keyword";
+import { Hash } from "@siteimprove/alfa-hash";
+import { Parser } from "@siteimprove/alfa-parser";
+import { Err, Ok, Result } from "@siteimprove/alfa-result";
+import { Slice } from "@siteimprove/alfa-slice";
+
+import { Token } from "../../syntax/token";
+
+const { either, left, pair, peek, option, right, separatedList } = Parser;
+
+export class Rectangle extends Value<"shape"> {
```

Comment:
Not really a "shape" per se. But the plan is to group it with the shapes for `clip-path`, so I guess it makes sense.